ProGetAutomation/Functions/Invoke-ProGetRestMethod.ps1

```diff
@@ -32,7 +32,11 @@ function Invoke-ProGetRestMethod
 
         [Switch]
         # Send the request as JSON. Otherwise, the data is sent as name/value pairs.
-        $AsJson
+        $AsJson,
+        
+        # Send File as InFile parameter

```

Comment:
Change documentation to read `Sends the contents of the file at this path as the body of the web request.`

Reply:
Done